app/assets/stylesheets/fonts.scss

```diff
@@ -1,19 +1,43 @@
 $brand: "Comfortaa", "Helvetica", sans-serif;
+$body-text: "Fira Sans", "Helvetica", sans-serif;
 
 .monospace{
   font-family: "Fira Mono", monospace;
   font-size: 1.1rem;
 }
 
+.brand {
+  font-family: $brand;
+}
+
+
+
 html {
-  font-family: "Fira Sans", "Helvetica", sans-serif;
+  font-family: $body-text;
 }
 
-.brand {
-  font-family: $brand;
+h1 {
+  @extend .brand;
+  text-align: center;
+}
+h2 {
+  @extend .dark-text;
+}
+h3 {
+  @extend .dark-text;
+}
+h4{
+  @extend .dark-text;
+}
+h5{
+
+}
+h6{
+  
 }
 
 p {
+  @extend .dark-text.secondary;
```

Comment:
Hey, sorry about the slow reply. BT internet outage nationwide hit right at that moment!

The MD spec suggests varying degrees of opaque black to allow better visibility of black on coloured backgrounds. I'm all for that. But I strongly prefer `color: rgba(0, 0, 0, 0.87)` over `opacity: 0.87`, especially on block elements like `p` because they're likely to contain other non-black elements (buttons, links etc.) that are unlikely to have their own opacity set.

I think in normal usage, paragraphs would be primary not secondary text?